src/schemas/db/userSchema.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file userSchema.js - Mongoose schema for users.
+ */
+const { Schema } = require('mongoose');
+const { ObjectID } = Schema.Types;
+
+const userSchema = new Schema(
+  {
+    email: {
+      type: String,
+      required: true
+    },
+
+    password: {
+      type: String,
+      required: true
+    },
+
+    linkedCompanyId: {
+      type: ObjectID
+    },
```

Comment:
Let's be more explicit here:

```suggestion
    linkedCompanyId: {
      type: ObjectID,
      required: false
    },
```